line.go

```diff
@@ -0,0 +1,97 @@
+package xinvoice
+
+import (
+	"github.com/invopop/gobl/bill"
+)
+
+// Line defines the structure of the IncludedSupplyChainTradeLineItem in the CII standard
+type Line struct {
+	ID              string           `xml:"ram:AssociatedDocumentLineDocument>ram:LineID"`
+	Name            string           `xml:"ram:SpecifiedTradeProduct>ram:Name"`
+	NetPrice        string           `xml:"ram:SpecifiedLineTradeAgreement>ram:NetPriceProductTradePrice>ram:ChargeAmount"`
+	TradeDelivery   *Quantity        `xml:"ram:SpecifiedLineTradeDelivery>ram:BilledQuantity"`
+	TradeSettlement *TradeSettlement `xml:"ram:SpecifiedLineTradeSettlement"`
+}
+
+// Quantity defines the structure of the quantity with its attributes for the CII standard
+type Quantity struct {
+	Amount   string `xml:",chardata"`
+	UnitCode string `xml:"unitCode,attr"`
+}
+
+// TradeSettlement defines the structure of the SpecifiedLineTradeSettlement of the CII standard
+type TradeSettlement struct {
+	ApplicableTradeTax []*ApplicableTradeTax `xml:"ram:ApplicableTradeTax"`
+	Sum                string                `xml:"ram:SpecifiedTradeSettlementLineMonetarySummation>ram:LineTotalAmount"`
+}
+
+// ApplicableTradeTax defines the structure of ApplicableTradeTax of the CII standard
+type ApplicableTradeTax struct {
+	TaxType        string `xml:"ram:TypeCode"`
+	TaxCode        string `xml:"ram:CategoryCode"`
+	TaxRatePercent string `xml:"ram:RateApplicablePercent"`
+}
+
+func newLine(line *bill.Line) *Line {
+	if line.Item == nil {
+		return nil
+	}
+	item := line.Item
+
+	lineItem := &Line{
+		ID:       item.Name,
+		Name:     item.Name,
+		NetPrice: item.Price.String(),
+		TradeDelivery: &Quantity{
+			Amount:   line.Quantity.String(),
+			UnitCode: string(item.Unit.UNECE()),
+		},
+		TradeSettlement: newTradeSettlement(line),
+	}
+
+	return lineItem
+}
+
+func newTradeSettlement(line *bill.Line) *TradeSettlement {
+	var applicableTradeTax []*ApplicableTradeTax
+	for _, tax := range line.Taxes {
+		tradeTax := &ApplicableTradeTax{
+			TaxType: tax.Category.String(),
```

Comment:
There is likely to be a specific list of types and not necessarily the same as the GOBL Categories. Most likely there needs to be a mapping here. Best bet would be to build a constructor for the `ApplicableTradeTax`, and try to make a mapping there.

Reply:
According to the XRechnung documentation the only acceptable value here at the moment is VAT.